machine-learning/solutions/seismic_salt_detection/trainer/task.py

```diff
@@ -50,45 +50,40 @@ def initialise_params():
     args_parser.add_argument(
         '--training_path',
         help='Location to training data.',
-        default='gs://energyforecast/data/csv/MLDataTrain.csv'
+        default='gs://mybucket/salt_segmentation/data/csv/train.csv'
     )
     args_parser.add_argument(
         '--validation_path',
         help='Location to validation data.',
-        default='gs://energyforecast/data/csv/MLDataValid.csv'
-    )
-    args_parser.add_argument(
-        '--mean_path',
-        help='Location to pre-computed means.',
-        default='gs://energyforecast/data/pickle/mean.pkl'
+        default='gs://mybucket/salt_segmentation/data/csv/valid.csv'
     )
     args_parser.add_argument(
-        '--std_path',
-        help='Location to pre-computed standard deviations.',
-        default='gs://energyforecast/data/pickle/std.pkl'
+        '--image_path',
+        help='Location of image folder.',
+        default='gs://mybucket/salt_segmentation/train/images/'
     )
     args_parser.add_argument(
-        '--dropout',
-        help='Dropout probability.',
-        default=0.2,
+        '--depth_mean',
+        help='Mean depth on training data.',
+        default=507.904,
```

Comment:
It seems dangerous to me to have a default mean and standard deviation.  I want this library's user to know that there's normalization that he or she has to configure.

What are your thoughts?

Reply:
I'm adding a warning about this to the README